tasks/installer/default.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: create install script temp file path
+  set_fact:
+    docker_install_script_path: '{{ docker_download_dir }}/ansible-docker-{{ ansible_user_uid }}-{{ ansible_date_time.epoch }}-{{ (2**48) | random }}'
+  run_once: true
```

Comment:
We can't use `run_once` here.  `docker_download_dir`, `ansible_user_uid` and `ansible_date_time.epoch` are usually different on the managed nodes.